crypto/fipsmodule/ec/p384.c

```diff
@@ -321,7 +321,9 @@ static void p384_point_double(p384_felem x_out,
 
   // z' = (y + z)^2 - gamma - delta
   // The following calculation differs from that in p256.c:
-  // An add is replaced with a sub in order to save 5 cmovznz.
+  // an add is replaced with a sub. This saves us 5 cmovznz operations
+  // when Fiat-crypto implementation of felem_add and felem_sub is used,
+  // and also a certain number of intructions when s2n-bignum is used.
```

Comment:
we can but it's not a single number because we have different implementations (x86 and arm) and also different assembly instructions are used so it's a bit more complicated. I think there is no reason to complicate the comment because we added it only to justify why we deviate from the implementation in `p256.c`. Saving some number of instructions is already a justification in my opinion, but if you insist I can count the exact number.